kong/db/init.lua

```diff
@@ -411,8 +411,18 @@ do
   local MigrationHelpers = require "kong.db.migrations.helpers"
   local MigrationsState = require "kong.db.migrations.state"
 
+
+  local last_schema_state
+
+
   function DB:schema_state()
-    return MigrationsState.load(self)
+    last_schema_state = MigrationsState.load(self)
```

Comment:
The `MigrationsState.load` may return more than one return value. Do we care about them? Perhaps better error reporting / logging?

Reply:
Excellent catch! Fixed!